handbook/data-stores/data-stores.md

```diff
@@ -0,0 +1,89 @@
+# MongoDB
+
+MongoDB is used by the majority of SystemLink services as the primary datastore. MongoDB Community Edition is included in the default installation of SystemLink will launch when the SystemLink server boots.
+
+!!!note
+    By default SystemLink stores all data on databases and the file system hosted on the SystemLink application server. This is referred to as a *single node* deployment. To improve the reliability of data storage and scale to larger volumes of data, remote data stores are recommended for all production deployments. This is referred to as a *multi node* deployment.
```

Comment:
> By default SystemLink stores all data on databases and the file system hosted on the SystemLink application server.

By default, both the databases and the file system are hosted on the server, right? Right now it reads like only the file system is. I plan to do a sentence-level review on my next round but wouldn't want to introduce an error here.

Reply:
Yes, the idea here is to point out that all file storage and all databases are on the same server as SystemLink by default. This isn't our recommended approach for production deployments.  I agree this sentence needs to be improved to clarify that.